--- libxaw3dxft-1.3.3/Makefile.am~	2012-02-05 16:38:19.000000000 +0100
+++ libxaw3dxft-1.3.3/Makefile.am	2012-02-27 11:35:54.428339579 +0100
@@ -38,6 +38,7 @@
 nodist_libXaw3dxft_la_SOURCES = $(BUILT_SOURCES)
 libXaw3dxft_la_includedir=$(includedir)/X11/Xaw3dxft
 libXaw3dxft_la_include_HEADERS = $(BASE_HDRS) $(EXTRA_INSTALL_HDRS)
+libXaw3dxft_la_LIBADD = -lXmu -lXt -lXft -lXpm
 nodist_libXaw3dxft_la_include_HEADERS = Xaw3dP.h Xaw3dXft.h
 CLEANFILES = $(BUILT_SOURCES)
 EXTRA_DIST = laygram.y laylex.l app-defaults $(ORPHANS)
